target_chains/ethereum/contracts/forge-test/Pyth.WormholeMerkleAccumulator.t.sol

```diff
@@ -167,6 +207,32 @@ contract PythWormholeMerkleAccumulatorTest is
         assertPriceFeedMessageStored(priceFeedMessages1[1]);
         assertPriceFeedMessageStored(priceFeedMessages2[0]);
         assertPriceFeedMessageStored(priceFeedMessages2[1]);
+
```

Comment:
There should be no difference between what is stored through `updatePriceFeeds` and what is returned by `parsePriceFeeds` if the updateData is the same aside from the following situation:

The reason that i'm only checking `priceFeedMessage2[1]` is b/c since both `priceFeedMessage1` and `priceFeedMessage2` are passed in the same `updateData` to `updatePriceFeeds` which should update the `priceFeed` information stored based on the latest `publishTime` per `priceFeedMessage.id`.
`parsePriceFeedUpdates` only parses the first occurrence per `priceFeedMessage.id` per `updateData` which is why i split the `updateData` & `priceIds` that i'm passing into `parsePriceFeedUpdates` and checking against.


This test originally was checking the implementation of `updatePriceFeeds` so I do still want to checks from `assertParsedPriceFeedStored`
I'll add comments to clarify these as well as moving the `parsePriceFeedsUpdates` prior to `updatePriceFeeds`

Reply:
just for background context, this behavior difference between parsePriceFeedUpdates & updatePriceFeeds was what I was referring to on slack yesterday